content/html/concepts/attributes/attributes.md

```diff
@@ -13,22 +13,39 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Attributes can be added to the opening tag of an HTML element to change its default behavior or provide additional data about it.
+HTML attributes are data that can be added to the opening tag of an HTML element. These attributes modify the default behavior of the element or provide additional information about it. 
+Attributes consist of a name and are usually set equal to a value. Values are enclosed in double or single quotation marks. 
 
-Attributes consist of a name and a value in the format `name="value"`:
+## Syntax 
 
-```pseudo
-<tag name="value"></tag>
+The syntax for adding attributes to an HTML element is as follows: 
+
+ ```html 
+ <tagname attribute="value"></tagname> 
+ ```
+
+> **Note:** Attribute names are not case-sensitive, but it is good practice to write HTML markup in lowercase.
+ 
+## Examples 
+
+ ### Image Attributes
+
+Using an  `<img>` element, which is commonly used to display images, reveals several **attributes**, such as `src`, `width`, and `height`. These attributes exert significant influence over the manner in which the image appears on the web page. The dimensions of the element in particular, once specified, enable precise control over the end result.
+
+```html
+<img src="source_of_image.png" width="500" height="600" />
 ```
 
-HTML elements can have any number of attributes.
+![GIF showcasing effect of height attribute on an img element](https://raw.githubusercontent.com/Codecademy/docs/main/media/changing-height-attribute-of-html-element.gif)
+
+
+ ###  `disabled` Attribute 
 
-The example below displays an image element with three attributes:
+The `disabled` attribute is applied to form elements like buttons and inputs to make them non-interactable:
 
 ```html
-<img src="nyan_cat.gif" width="500" height="600" />
+<button disabled>Submit</button>
 ```
 
-These declare its source, width, and height.
+![Disable attribute on a <button> element](https://raw.githubusercontent.com/Codecademy/docs/main/media/media/button-disabled-attribute.gif)
 
```

Comment:
[nitpick] While showing the output is helpful, adding introductory text that indicates this would clarify the example more. 
```suggestion
This is the output of the above code snippet: 
![Disable attribute on a <button> element](https://raw.githubusercontent.com/Codecademy/docs/main/media/media/button-disabled-attribute.gif)

```